src/pal/src/libunwind/src/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 # The HAVE___THREAD set to 1 causes creation of thread local variable with tls_model("initial-exec")
 # which is incompatible with usage of the unwind code in a shared library.
 add_definitions(-DHAVE___THREAD=0)
-add_definitions(-DHAVE_CONFIG_H=1)
+add_definitions(-DHAVE_CONFIG_H)
```

Comment:
This is a setting for a third party library whose source we include. Can you please revert the change here?